jpmml-migration-recipe/src/test/java/org/kie/openrewrite/recipe/jpmml/JPMMLVisitorTest.java

```diff
@@ -0,0 +1,157 @@
+package org.kie.openrewrite.recipe.jpmml;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.Properties;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.java.tree.J;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.kie.openrewrite.recipe.jpmml.CommonTestingUtilities.getCompilationUnitFromClassSource;
+import static org.kie.openrewrite.recipe.jpmml.CommonTestingUtilities.getExecutionContext;
+import static org.kie.openrewrite.recipe.jpmml.CommonTestingUtilities.getMethodInvocationFromClassSource;
+import static org.kie.openrewrite.recipe.jpmml.CommonTestingUtilities.getNewClassFromClassSource;
+import static org.kie.openrewrite.recipe.jpmml.JPMMLVisitor.TO_MIGRATE_MESSAGE;
+
+class JPMMLVisitorTest {
+
+    private JPMMLVisitor jpmmlVisitor;
+    private static Properties CHANGED_INSTANTIATIONS;
+
+    @BeforeAll
+    public static void setup() throws IOException {
+        try (InputStream input = JPMMLRecipe.class.getResourceAsStream ("/changed_instantiation.properties")) {
```

Comment:
Why loading this resource from a property file? I think this configuration information is supposed to be in rewrite.yml file.